CONTRIBUTING.md

```diff
@@ -0,0 +1,205 @@
+  - [10. Non-code contributions](#10-non-code-contributions)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+</div>
+
+---
+
+## How to open a GitHub issue & file a bug report
+
+### Working on a new feature or filing a bug report
+
+If you would like to add a new feature or file a new bug report, we prefer that you open a new issue on the Rasa repository before creating a pull request.
+
+It’s important to note that when opening an issue, you should first do a quick search of existing issues to make sure your suggestion hasn’t already been added as an issue.
+If your issue doesn’t already exist, and you’re ready to create a new one, make sure to state what you would like to implement, improve or bugfix. We have provided a template to make this process easier for you.
+
+**To open a Github issue, go to the RasaHQ repository, select “Issues”, “New Issue” then “Feature Request” or “Bug Report” and fill out the template.**
+
+![](https://www.rasa.com/assets/img/contributor-guidelines/opening-new-issue.png)
+
+The Rasa team will then get in touch with you to discuss if the proposed feature aligns with the company's roadmap, and we will guide you along the way in shaping the proposed feature so that it could be merged to the Rasa codebase.
+
+### Working on an existing feature
+
+If you choose to work on an existing issue, check out the Rasa contributors board to find existing open issues you can work on.
+
+The issues are handpicked by the Rasa team to have labels which correspond to the difficulty/estimated time needed to resolve the issue.
+
+**To work on an existing issue, go to the contributor project board, add a comment stating you would like to work on it and include any solutions you may already have in mind.**
+
+![](https://www.rasa.com/assets/img/contributor-guidelines/exiting-issue-sara.png)
+
+Someone from Rasa will then assign that issue to you and help you along the way.
+
+---
+
+## How to open a GitHub Pull Request
+
+### What is a Pull Request (PR)?
+
+This is how the GitHub team defines a PR:
+
+> “Pull requests let you tell others about changes you’ve pushed to a branch in a repository on GitHub. Once a pull request is opened, you can discuss and review the potential changes with collaborators and add follow-up commits before your changes are merged into the base branch.”
+
+This process is used by both Rasa team members and Rasa contributors to make changes and improvements to Rasa Open Source.
+
+### What to know before opening a PR
+
+#### Opening issues before PRs
+
+We usually recommend opening an issue before a pull request if there isn’t already an issue for the problem you’d like to solve. This helps facilitate a discussion before deciding on an implementation. See How to open a GitHub issue & file a bug report.
+
+#### Draft PRs
+
+To get some quick initial feedback from the Rasa team, you can create a draft pull request.
```

Comment:
Mm true, but can we make sure they don't open it at least until they're ready for feedback? Cause if they open it before (e.g. https://github.com/RasaHQ/rasa/pull/5129#issuecomment-579495997) it can be less than idea to have it sitting there for a while